### PR TITLE
Translate System Menus

### PR DESCRIPTION
**Pull request by stevietv**
#### Database Migration
NO

#### Description
add translations to all strings within the System menu/submenus

### CODE REVIEW

File: frontend/src/System/Logs/Files/LogFiles.js [hunk 1]
```diff
@@ -76,13 +77,13 @@ class LogFiles extends Component {
         <PageContentBody>
           <Alert>
             <div>
-              Log files are located in: {location}
+              {translate('LogFilesLocation')}: {location}
```
Comment (markus101) [inline frontend/src/System/Logs/Files/LogFiles.js:80]: `location` should be part of the translation so languages that prefer a different syntax can translate it properly.

File: frontend/src/System/Status/About/About.js [hunk 2]
```diff
@@ -30,54 +31,54 @@ class About extends Component {
...
             data={version}
           />
 
           {
             packageVersion &&
               <DescriptionListItem
-                title="Package Version"
-                data={(packageAuthor ? <span> {packageVersion} {' by '} <InlineMarkdown data={packageAuthor} /> </span> : packageVersion)}
+                title={translate('PackageVersion')}
+                data={(packageAuthor ? <span> {packageVersion} {translate('by')} <InlineMarkdown data={packageAuthor} /> </span> : packageVersion)}
```
Comment (markus101) [inline frontend/src/System/Status/About/About.js:45]: Similar thing here, though this is more complicated with the HTML.

File: frontend/src/System/Status/About/About.js [hunk 3]
```diff
@@ -30,54 +31,54 @@ class About extends Component {
...
               />
           }
 
           {
             isNetCore &&
               <DescriptionListItem
-                title=".Net Version"
+                title={translate('NetCore')}
```
Comment (markus101) [inline frontend/src/System/Status/About/About.js:52]: Core isn't part of the official name any more.

**Review by markus101 — CHANGES_REQUESTED** [review]

File: frontend/src/System/Status/About/About.js (hunk 3 above)
Comment (stevietv) [reply frontend/src/System/Status/About/About.js]: i used NetCore to be in line with radarr, but adjusted to DotNetVersion instead.

File: frontend/src/System/Status/About/About.js (hunk 2 above)
Comment (stevietv) [reply frontend/src/System/Status/About/About.js]: So for this I went with translating the whole thing and if a packageAuthor exists wrapping it all in an `InlineMarkdown`.

File: frontend/src/System/Status/MoreInfo/MoreInfo.js [hunk 4]
```diff
@@ -12,52 +13,70 @@ class MoreInfo extends Component {
...
           <DescriptionListItemDescription>
             <Link to="https://discord.sonarr.tv/">discord.sonarr.tv</Link>
           </DescriptionListItemDescription>
 
-          <DescriptionListItemTitle>IRC</DescriptionListItemTitle>
+          <DescriptionListItemTitle>
+            {translate('IRC')}
+          </DescriptionListItemTitle>
           <DescriptionListItemDescription>
             <Link to="irc://irc.libera.chat/#sonarr">#sonarr on Libera</Link>
```
Comment (markus101) [inline frontend/src/System/Status/MoreInfo/MoreInfo.js:57]: I think we could translate the link text as well

**Review by markus101 — APPROVED** [review]

File: frontend/src/System/Status/MoreInfo/MoreInfo.js (hunk 4 above)
Comment (stevietv) [reply frontend/src/System/Status/MoreInfo/MoreInfo.js:57]: added in #5823

**Pull request merged**